changelog.asc

```diff
@@ -2,6 +2,76 @@
 :sectanchors:
 = AnkiDroid Changelog
 
+== Version 2.16.0 (20230501)
+* You waited patiently nearly 3 years for this release! ⌛ Thank you
+* What an effort! v2.16 contains 2,693 changes from 143 different authors 🤯
+* AnkiDroid is a labor of ❤️ with a mission to help the whole world learn more
+* https://opencollective.com/ankidroid[you can ❤️ us back so we have more time to work on it]
+* “Why It Took So Long”: Scoped Storage replaces sdcard Storage
+** Faster syncing and importing
+** AnkiDroid no longer needs permission to access “All Files”!
+** ...But you lose access to data on uninstall
+** Sync, export/import, check media/database disabled during migration
+** Manage Space Activity in System Settings
+** Added “Backup Reminder” prompt
+** https://github.com/ankidroid/Anki-Android/wiki/Storage-Migration-FAQ[More info in our FAQ]
+* A huge number of crash fixes, too many to mention
+* “Auto” Day/Night theming to follow system, Android 13+ themed icon
+* Added Anki “New Anki Backend option in advanced settings - will be default in 2.17
+** We include anki 2.1.61 now, we’re fully up to date!
+** Stats Heatmap
+** Scheduler v3 support
+** CSV Import, Text import
+** Card Browser Searches
+** Card Info and Deck Options from upstream
+* New gesture manager, supports gamepad and bluetooth keyboard
+* Translations, new languages, lots of fixes, https://crowdin.com/project/ankidroid/[you can help!]
+* Add support for hierarchical tags
+* New Onboarding Screen
+* New Preferences
+** Icons & Better Categories
+** Increase max possible backups to 99
+** Full screen navigation drawer option
+** Add Preference for double-tap time interval
+** “Insert Field” in Card Template Editor option
+** Add "Allow sync on metered connections" option
+** New “About” section
+** “Developer Options” is now available
+* Flags added: Pink, turquoise & purple flags as in Desktop
+* Drawing Activity
+* Custom Sync Server improvements
+** Updated to match new Anki Desktop built-in sync server abilities
+** Use different server for collection versus media
```

Comment:
I believe that got reverted

Reply:
oh yeah - I think you're right. It was a big part of the original commit series from oakkitten but when you came and reorganized things you re-combined